tests/xharness/Jenkins/TestTasks/JenkinsTestTask.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Diagnostics;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.DotNet.XHarness.iOS.Shared.Logging;
+using Xharness.TestTasks;
+
+namespace Xharness.Jenkins.TestTasks {
+	public abstract class JenkinsTestTask : Xharness.TestTasks.TestTasks
```

Comment:
In hindsight naming anything with `Jenkins` was a bad idea (because this is not limited to Jenkins at all) - but that doesn't mean we have to make the mistake bigger :smile:

```suggestion
	public abstract class AppleTestTask : Xharness.TestTasks.TestTasks
```